app/helpers/application_helper.rb

```diff
@@ -424,7 +424,8 @@ def db_to_controller(db, action = "show")
         "ManageIQ::Providers::CloudManager::CloudDatabase",
         "ManageIQ::Providers::ConfigurationManager",
         "ManageIQ::Providers::AnsibleTower::AutomationManager::ConfiguredSystem",
-        "ManageIQ::Providers::AnsibleTower::AutomationManager::Job", "ConfigurationScript"
+        "ManageIQ::Providers::ExternalAutomationManager::OrchestrationStack"
+        "ConfigurationScript"
```

Comment:
There's a bug here - there should be a comma or something?

```suggestion
        "ManageIQ::Providers::ExternalAutomationManager::OrchestrationStack",
        "ConfigurationScript"
```

Reply:
I'm surprised this passed through testing?

Reply:
Well you're right this was 100% not intentional but I just double checked and it still does work and doesn't throw a syntax error :shrug: 

```
?> def foo(bar)
?>   case bar
?>   when "a", "b", "c"
?>     "1"
?>   when "e", "f"
?>     "g"
?>     "2"
?>   end
>> end
=> :foo
>> foo("a")
=> "1"
>> foo("b")
=> "1"
>> foo("f")
=> "2"
```

Learned something new, anyway I'll fix it now.